.github/workflows/deploy_sam_app.yml

```diff
@@ -22,6 +22,13 @@ jobs:
       - name: Checkout Branch
         uses: actions/checkout@v2
 
+      - name: Yarn install and build
```

Comment:
A few comments:

1.  We should use the same capitalization as the other steps.  So we should either change this to "Yarn Install and Build," or change the capitalization of the other steps to only capitalize the first word.
2. Is the last command `cd ..` necessary, or does GitHub automatically start the next step in the working directory?
3. I think we should leave the `deploy_sam_app` workflow as it is and add this step to a new workflow called `deploy_node_sam_app`.

Reply:
Maybe it would be cleaner to use the --cwd argument instead of changing the working directory? What do you think?

Reply:
Updated the capitalization of the step name and also made the step conditional based on the `node` input passed in. You were right, the `cd ..` is not necessary, so I removed that. I have no preference for whether to do the `cd` or use `--cwd`, but given that we don't need to navigate back, I don't see one being cleaner than the other.